python/vyos/firewall.py

```diff
@@ -337,6 +337,15 @@ def parse_rule(rule_conf, fw_name, rule_id, ip_name):
             target = rule_conf['jump_target']
             output.append(f'NAME{def_suffix}_{target}')
 
+        if 'queue' in rule_conf['action']:
+            if 'queue' in rule_conf:
+                target = rule_conf['queue']
+                output.append(f'num {target}')
+
+            if 'queue_options' in rule_conf:
+                queue_opts = ','.join(rule_conf['queue_options'])
```

Comment:
`num` is actually needed here. Despite ruleset uses the word `to`, if using `to` while processing, when loading config, we get an error.. So `num` seems to be needed
```
vyos@vyos # cat /run/nftables.conf
....
...
    chain NAME_FOO {
        counter drop comment "FOO-1"
        counter queue num 2-5 bypass comment "FOO-2"             ##  --> Works OK
#        counter queue to 2-5 bypass comment "FOO-2"             ## --> Does not work. Error while loading
        counter drop comment "FOO default-action drop"
    }
```